data/transactions/logic/simulate_test.go

```diff
@@ -526,6 +594,13 @@ func TestSignatureCheck(t *testing.T) {
 	require.NoError(t, err)
 	require.Empty(t, result.FailureMessage)
 	require.Empty(t, result.SignatureFailureMessage)
+
+	// should error with invalid signature
+	txgroup[0].Sig[0] = 0x0 // != 0xe
```

Comment:
Is the first byte always `0xe`?

If not, I suggest doing something like this, which will always work for any bytestring:

https://github.com/algorand/go-algorand/blob/7a56ea79e4a15c8195b426380643254839949a24/data/transactions/logic/evalCrypto_test.go#L367-L369